src/GovUk.Education.ExploreEducationStatistics.Publisher/Functions/GenerateReleaseContentFunction.cs

```diff
@@ -48,7 +49,12 @@ public class GenerateReleaseContentFunction
             await UpdateStage(message, Started);
             try
             {
-                var context = new PublishContext(GetNextScheduledPublishingTime(), true);
+                var publishStagedReleasesCronExpression = Environment.GetEnvironmentVariable("PublishReleaseContentCronSchedule") ?? "";
+                var nextScheduledPublishingTime = CrontabSchedule.Parse(publishStagedReleasesCronExpression, new CrontabSchedule.ParseOptions
+                {
+                    IncludingSeconds = CronExpressionHasSecondPrecision(publishStagedReleasesCronExpression)
+                }).GetNextOccurrence(DateTime.UtcNow);
```

Comment:
I think we should do yes for consistency, but I didn't want to change the Publisher to use it as part of this change in case it introduces a bug in `nextScheduledPublishingTime` since this function is not unit tested currently.

I will create a new Jira ticket to suggest we change it and change it in the Admin as well where we're using it for the memory caching.